ros/src/system/autoware_health_checker/src/health_aggregator.cpp

```diff
@@ -0,0 +1,114 @@
+#include <autoware_health_checker/health_aggregator.h>
+
+HealthAggregator::HealthAggregator(ros::NodeHandle nh,ros::NodeHandle pnh)
+{
+    nh_ = nh;
+    pnh_ = pnh;
+}
+
+HealthAggregator::~HealthAggregator()
+{
+
+}
+
+void HealthAggregator::run()
+{
+    system_status_pub_ = nh_.advertise<autoware_system_msgs::SystemStatus>("/system_status",10);
+    node_status_sub_ = nh_.subscribe("/node_status",10,&HealthAggregator::nodeStatusCallback,this);
+    diagnostic_array_sub_ = nh_.subscribe("/diagnostic_agg",10,&HealthAggregator::diagnosticArrayCallback,this);
```

Comment:
I think that it is a specification that receives the same topic from many nodes, but is it not necessary to set the queue size to be large or to be changed by the parameter?

Reply:
I think it will be no problem.